packages/eds-tokens-build/src/index.ts

```diff
@@ -228,7 +278,7 @@ StyleDictionary.registerTransform({
 
 const cssTransforms = [
   'name/cti/kebab',
-  'eds/css/spacing/shorthand',
+  //'eds/css/spacing/shorthand',
```

Comment:
What remains to make this work?

Reply:
As discussed, we'll find a solution for this later as seperate tokens  (if we do it at all)